readme/apps/keyboard_interface.md

```diff
@@ -0,0 +1,48 @@
+# Keyboard interface
+
+This document includes tips for using Joplin from either:
+- a keyboard-only device (desktop)
+- a screen reader (desktop and/or mobile)
+
+## Navigating the desktop app
+
+Joplin's UI is divided into regions. In addition to the standard screen reader "jump to" shortcuts, Joplin includes its own shortcuts for navigating to these regions.
+
+See items in the "Go"/"Focus" section of the menubar for keyboard shortcuts to move focus directly to some of these regions.
+
+### Accessibility regions
+
+Joplin's main window is divided into three main regions:
+1. **Navigation 1**: The sidebar.
+2. **Navigation 2**: The notes list.
+3. **Main content**: The editor and viewer.
+
+Within the **main content**, Joplin has:
+1. **Editable**: A note title input.
+2. Two toolbars.
+3. **Editable**: A note editor.
+4. **Frame**: A note viewer.
+
+Many screen readers include shortcuts for jumping to the next/previous of each of these roles.
+
+### Finding notes, tags, and notebooks
+
+It can sometimes be difficult to find a tag/note/notebook using Joplin's navigation sidebars. In these cases, the ["Go to Anything" dialog](./search.md#goto-anything) can simplify accessing an item.
+
+It's also possible to find notes using the search in the sidebar. By default, pressing <kbd>F6</kbd> moves focus to this search entry.
+
+### Toggling tab-key navigation
+
+By default, pressing <kbd>tab</kbd> indents, rather than moves focus. This can be changed by toggling "Tab moves focus" from the "View" menu. By default, this can also be toggled by pressing <kbd>ctrl</kbd>-<kbd>m</kbd>.
+
+## Navigating the mobile app
```

Comment:
This section needs more information. Here are a few things it could include:
- How to select notes (then duplicate/move/delete them) using a screen reader.
- How to edit a notebook's title/location using a screen reader.
- How to toggle to-dos.
- Links to documentation for customizing the toolbar.
- Links to voice typing documentation (Android).
- Link to audio recording documentation (this may need to be created).
- Perhaps also discuss known accessibility issues (though better would be to fix these :) ).

Reply:
I think giving an overview for now may be good enough? Then the document can be completed over time based on user feedback. The drawback of having a lot of information is that it may become stale and would have to be updated.